src/app/lib/config/services/polska.ts

```diff
@@ -388,20 +384,6 @@ export const service: DefaultServiceConfig = {
         'BBC. BBC nie ponosi odpowiedzialności za treści na stronach zewnętrznych.',
     },
     timezone: 'Europe/Warsaw',
-    navigation: [
-      {
-        title: 'News',
-        url: '/polska',
-      },
```

Comment:
Why don't we keep this - it will just redirect us back to the About Us page - right?

Reply:
It would do , but would that be quite an annoying ux - clicking something labelled news but it just keeps taking you to a non news page, could look broken?

Reply:
Is it worth checking with Product? Also, are there any concerns about not having a navigation bar on the site at all - is there anything we need to consider accessibility wise?

Reply:
I can check with neil on just keeping the news one in, interesting point about a11y though - i don't think it should have an effect as the skip to content link will still be in place but can check this too with catharine

Reply:
We may need to do a quick a11y swarm before go live if we decide to have no nav - just in case.

Reply:
neil says lets remove this one as well , and catharine says there are no a11y concerns 🥳  think we're okay to merge this as is if you're okay with that? ive done a quick check of the pages without the nav in place to check the skip to content link all works as expected and all seems okay

Reply:
Yes, sounds good! Thanks for following up with everyone. 🚢